release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -3608,8 +3608,6 @@ $ oc adm release info 4.12.15 --pullspecs
 [id="ocp-4-12-15-updating"]
 ==== Updating
 
-<<<<<<< HEAD
-=======
```

Comment:
Including a fix for https://docs.openshift.com/container-platform/4.12/release_notes/ocp-4-12-release-notes.html#ocp-4-12-15